grakn-engine/src/main/java/ai/grakn/engine/postprocessing/util/TaskConfigReader.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.engine.postprocessing.util;
+
+import ai.grakn.concept.ConceptId;
+import ai.grakn.engine.tasks.TaskConfiguration;
+import ai.grakn.util.REST;
+import ai.grakn.util.Schema;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * <p>
+ *     Helper class to extract details from commit log
+ * </p>
+ *
+ * <p>
+ *     This class is used to extract information from Task configs which is needed by
+ *     {@link ai.grakn.engine.postprocessing.PostProcessingTask},
+ *     {@link ai.grakn.engine.postprocessing.UpdatingInstanceCountTask} and
+ *     {@link ai.grakn.engine.loader.MutatorTask}
+ * </p>
+ *
+ * @author fppt
+ */
+public class TaskConfigReader {
```

Comment:
I don't think these functions should be in a separate class. Other than the "keyspace" one, they are only used in one place. And Its not necessary that the user places the keyspace in the root of the configuration object. 

Even if we do keep the class, we should remove `getCountUpdatingJobs` and `getPostProcessingJobs`, which are only used in their respective tasks.